### PR TITLE
💡 Add light mode

### PR DESCRIPTION
**Pull request by B2o5T**
fixes https://github.com/the-guild-org/the-guild-website/issues/544

### CODE REVIEW

**Comment by dotansimha** [comment]
Thanks @B2o5T 
looks good, only issue I see is a weird flickering when switching pages (I see it as dark mode for 1/2s and then it switches to light)

**Comment by B2o5T** [comment]
@dotansimha yes! Saw it issue also, this is because header don’t use client side routing

File: pages/_app.tsx [hunk 1]
```diff
@@ -56,7 +100,6 @@ export default class MyApp extends App {
         <Head>
           <meta name="viewport" content="width=device-width, initial-scale=1" />
           <meta charSet="utf-8" />
-          <link rel="alternate" type="application/rss+xml" href="/feed.xml" />
```
Comment (B2o5T) [inline pages/_app.tsx:59]: Duplicate used in line 122

File: pages/blog/caching-data-with-dataloader.mdx [hunk 2]
```diff
@@ -82,7 +82,7 @@ type author {
 
 and the corresponding data type would look like this:
 
-```graphql
+```ts
```
Comment (B2o5T) [inline pages/blog/caching-data-with-dataloader.mdx:85]: Typo, it’s typescript code

File: lib/sitemap.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import { promises } from 'node:fs';
+import { writeFile } from 'node:fs/promises';
```
Comment (charlypoly) [inline lib/sitemap.ts:1]: @B2o5T I use the following next library on my personal website to generate an accurate sitemap: https://github.com/iamvishnusankar/next-sitemap

The sitemap below does not include the blog posts links

**Pull request merged**